hass/devices.js

```diff
@@ -212,6 +212,101 @@ module.exports = {
       }
     }
   ],
+  '5-1619-20549': [
+    // Intermatic PE653 MultiWave Receiver
+    {
+      type: 'climate',
+      object_id: 'pool_thermostat',
```

Comment:
Could you use more generic names? Is this device usually used to create this entities or is just for your use case?

Reply:
These are the names used by the device itself, see next comment.